main.nf

```diff
@@ -284,10 +474,66 @@ process fastp {
     }
 }
 
+/*
+ * Remove PhiX contamination from Illumina reads
+ * TODO: PhiX into/from iGenomes.conf?
+ */
+if(!params.keep_phix) {
```

Comment:
Couldn't we use the same process for both long and short reads? minimap2 (and nanolyse) should work fine enough with short reads, and will be much faster.

Reply:
minimap2 might be an option for short reads, I chose nanolyse for long reads since it is a specialized tool for this task. I am not insisting on bowtie2.